public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -175,6 +177,8 @@ export class EhrViewComponent implements OnInit, OnDestroy {
     this.searchResult.items = this.searchResult.items.filter(
       x => this.dbc.TO_SUPPRESS_PMS.indexOf(x.conceptId) === -1);
     this.items = this.searchResult.items;
+    this.matchType = this.searchResult.matchType.toString();
```

Comment:
nit: do you need these variables? Seems like you could just directly reference `this.searchResult.matchType` below